### PR TITLE
Add vale to install and lint

### PR DESCRIPTION
**Pull request by amcintosh**
Builds failing due to new lint errors.

See https://github.com/chasinglogic/taskforge/issues/20

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,6 +21,7 @@ install-dev:
 	$(PIP) install --editable .
 	$(PIP) install --editable ".[mongo]"
 	$(PIP) install -r requirements.dev.txt
+	go get github.com/errata-ai/vale
```
Comment (chasinglogic) [inline Makefile]: Seeing as this is a Python project I'm not sure it's reasonable to think everyone will have a go environment installed and configured.

Is there no way we can install the precompiled binary?

Comment (chasinglogic) [reply Makefile]: On second thought, I'm really ok with just leaving vale installation instructions as part of docs getting started and moving this line into the travis config.

Comment (amcintosh) [reply Makefile]: It's a good point. 
I could add it to the instructions or move it into the travis config. I could also move it into a separate `install-ci` Make target if you want to keep all the logic in the Makefile.

Which would you prefer?

Comment (chasinglogic) [reply Makefile]: For now just put it directly in the install section of the travis.yml, I'm working on contributing documentation now and I'll add a note about how to lint documentation with vale.

Thanks!

Comment (amcintosh) [reply Makefile]: Ok. Moved!

**Comment by chasinglogic** [comment]
lgtm thanks for your help!

**Pull request merged**